docs/README.md

```diff
@@ -1,170 +1,169 @@
 # Nori Smart Contracts
 
-Nori's product is the NRT or Nori _Removal_ Tonne. This can be understood as a carbon removal credit granted to a supplier for the CO2 they have removed from the atmosphere. The supplier consigns their NRTs to Nori's marketplace for sale and gets paid in ERC20 tokens. Ultimately, suppliers will be paid in the _NORI_ token, which will be exchangable 1:1 for 1 NRT and is designed to enable price discovery for carbon removal. Initially, however, suppliers will be paid in _USDC_ at a price set by Nori. A buyer of NRTs is minted a non-transferrable _Certificate_ (NCCR) which ultimately owns the sold NRTs.
+Nori's core product is the Nori _Removal_ Tonne (NRT). NRTs can be understood as carbon removal credits that are granted to a supplier for the CO2 they have removed from the atmosphere. For each tonne of carbon removed, a supplier is granted 1 NRT. Suppliers then consign their NRTs to Nori's marketplace, effectively listing them for sale at a rate of 1 NRT per 1 _NORI_ token (an [ERC20](https://github.com/ethereum/ethereum-org-website/blob/b17088f97b805dbc947e629d3ae5358ed27ac076/src/content/developers/docs/standards/tokens/erc-20/index.md) compatible token on [the Polygon blockchain](https://polygon.technology/)).
```

Comment:
similar content, but moved some stuff to a footnotes section. Open to undoing this bit if you feel strongly.

Reply:
`For each tonne of carbon removed, a supplier is granted 1 NRT.`  Maybe leave this part out since it's more nuanced?

Also this crossed with Amie's docs update to reflect the USDC changes.